bigquery/gcloud/aio/bigquery/bigquery.py

```diff
@@ -401,15 +467,18 @@ async def insert_via_query(
             self, query: str, session: Optional[Session] = None,
             write_disposition: Disposition = Disposition.WRITE_EMPTY,
             timeout: int = 60, use_query_cache: bool = True,
-            dry_run: bool = False) -> Job:
+            dry_run: bool = False, use_legacy_sql: bool = True) -> Job:
```

Comment:
Given we're deprecating this method, let's avoid adding the new `use_legacy_sql` functionality -- rather stop maintaining it entirely than have it be partially maintained, since that might be a bit confusing to users.